### PR TITLE
Switch all hex to bytes conversions to ethjs equivalents

### PR DESCRIPTION
**Pull request by alivia-g**
fixes https://github.com/ethereumjs/ultralight/issues/624

### CODE REVIEW

**Comment by acolytec3** [comment]
Thanks for the PR.  I'm not 100% sure what all you were working on here but this shouldn't require updating any dependencies (notably eslint and related plugins) and there should be no updates to docs.  This is purely changing the `fromHexString`/`toHexString` methods over to the equivalents from `@ethereumjs/util`.  We won't be able to take this in without removing all the extra stuff.

Notably, if you want to continue with this, it should be pretty close a simple "find/replace" across 3 main packages (`portalnetwork`, `cli`, `era`).

Let me know if you want to continue on this one.

**Comment by alivia-g** [comment]
I will continue working on this one, thank you for explaining.

**Comment by alivia-g** [comment]
Currently it's passing portalnetwork test and integration test, cli test still have 1 failed test. I'm not sure how to proceed from here.

Here is the failed cli test case:
```
 FAIL  test/rpc/portal/beaconLocalContent.spec.ts > portal_beaconLocalContent tests > should not find any local content
AssertionError: expected -32603 to equal -32009

- Expected
+ Received

- -32009
+ -32603

 ❯ test/rpc/portal/beaconLocalContent.spec.ts:20:12
     18|       ),
     19|     ])
     20|     assert.equal(res.error.code, -32009)
       |            ^
     21|     ultralight.kill(9)
     22|   }, 10000)
```

File: packages/cli/src/rpc/modules/portal.ts [hunk 1]
```diff
@@ -11,9 +11,8 @@ import {
   NodeLookup,
   PingPongCustomDataType,
   PortalWireMessageType,
-  fromHexString,
+  hexToBytes,
```
Comment (ScottyPoi) [inline packages/cli/src/rpc/modules/portal.ts:14]: `bytesToHex` and `hexToBytes` should be imported from `@etherumjs/util`

**Review by ScottyPoi — COMMENTED** [review]
Regarding the hex / bytes conversion functions -- 

`hexToBytes` and `bytesToHex` must be imported from `@ethereumjs/util`.  There are several files in which you are attempting to import these from other libraries.

any other changes (such as to `package.json`) should be reverted.  keep the scope of this PR limited to the conversion functions.

**Comment by acolytec3** [comment]
@alivia-g we updated `package-lock` so maybe the dev issues you were seeing should be resolved now.  Are you still interested in giving this a go?  If so, can you redo this PR along the lines @ScottyPoi described.  Basically, everywhere you see `toHexString` and `fromHexString` imported from `@chainsafe/discv5`, those imports should be replaced by `bytesToHex` and `hexToBytes` imported from `@ethereumjs/util`.

**Comment by acolytec3** [comment]
Closed due to lack of follow up

**Pull request closed without merging**